daml/Passage.daml

```diff
@@ -0,0 +1,38 @@
+module Passage where
+
+template LocalPassage with
```

Comment:
why there are two types of `Passage`?

Reply:
`LocalPassage` represents a passage which was created locally by some secondary server, and it is visible only to that server and the master/main node. After that, the master takes the `LocalPassage` contract, archives it, and creates a `GlobalPassage` contract in order to synchronize it for all of the servers. 
In the event of a disconnection from the network, the secondary server will be able to keep using the `LocalPassage` contracts that it created, and when it will reconnect, the master will synchronize those passages by replacing them with `GlobalPassage` contracts that are visible to everyone.

Reply:
Also, this same concept applies to the permits.